### ISSUE TITLE
Expand on trace_history_api

### ISSUE THREAD

**Issue by matthewdarwin** [eosrio/hyperion-history-api#34]
B1 added more features to trace history API, which should be available starting in 2.0.6 https://github.com/EOSIO/eos/pull/9047

Probably Hyperion needs to adjust to the new feature set?

**Comment by igorls** [comment]
Yes, some of those would require a reindex after updating the block mappings

**Comment by matthewdarwin** [comment]
Related: see also: https://github.com/EOSIO/eos/pull/9141

**Comment by igorls** [comment]
Closing — this issue references a version of Hyperion that has been superseded by v4.x. If you still experience this on the latest version, please open a new issue.
